elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -308,7 +308,9 @@ class PfeAccordion extends PFElement {
 
   _transitionEndHandler(evt) {
     const header = evt.target.previousElementSibling;
-    header.classList.remove("animating");
+    if (header) {
```

Comment:
should this branch be re-adding this

Reply:
`previousElementSibling` was throwing errors in IE so I couldn't confirm broadcast styles were working, by adding this it unblocked testing on this branch